proposals/tsc-voting-procedures.md

```diff
@@ -0,0 +1,199 @@
+# TSC Voting Procedures Proposal
+
+## Overview
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Voting Methods
+
+### 1. In-Meeting Votes
+- **When Used**: For decisions during TSC meetings with quorum present
+- **Threshold**: Simple majority of members present
+- **Recording**: Documented in meeting minutes with vote counts
+
+### 2. Asynchronous Votes via GitHub Pull Request
+- **When Used**: 
+  - When quorum cannot be achieved in meetings
+  - For decisions requiring extended review/discussion
+  - For charter-mandated votes (e.g., license exceptions)
+- **Location**: Pull requests in the `proposals/` directory
+- **Notification**: All TSC members must be notified via:
+  - GitHub PR mention (@username)
+  - Slack notification in TSC channel
+
+## Voting Process
```

Comment:
This is only the process / workflow for "Asynchronous Votes via GitHub Pull Request". Please make that clear and add a short section for the workflow "In-Meeting Votes"